src/test/java/showcase/neodymium/tests/data/dataobjects/ServiceTile.java

```diff
@@ -0,0 +1,23 @@
+package showcase.neodymium.tests.data.dataobjects;
+
+/**
+ * POJO class to map data set test data The class specification has to match the test data structure to perform the
+ * mapping.
+ */
+public class ServiceTile
+{
+    private String heading;
+
+    private String explanation;
+
+    public String getHeading()
+    {
+        return heading;
+    }
+
+    public String getExplanation()
+    {
+        return explanation;
+    }
+
```

Comment:
add a simple toString method for the data object so it's easier to display the objects content